README.md

```diff
@@ -1,117 +1,158 @@
+
+where `<rundir>` is an absolute path to the run directory.
 
 # Developing the model
 
 The `fv3gfs-environment` docker image is useful for development purposes where the model code will be recompiled repeatedly in an interactive fashion. To start a bash shell in a docker container with the FV3 source tree mounted at `/FV3`, run the command:
 
-    make enter
+```bash
+make enter
+```
 
 If necessary, this will build the image, but it will overwrite the compiled sources
 with a bind mount to your host filesystem. You will need to compile the model with
-the filesystem bind-mounted in this way. This can be done automatically with:
+the filesystem bind-mounted in this way. Once in the container, you can compile the model using the commands
 
-    make compile_dev
+To compile the model and generate an executable you can use the commands
+
+```bash
+cd /FV3
+make clean
+make -j8
+```
 
 # Testing the model
 
 Tests are included in the `tests` subdirectory. You can see which ones run in
-continuous integration by inspecting `.circleci/config.yml`. Regression tests are
+continuous integration by inspecting `.circleci/config.yml`.
+
+Regression tests which check the bit-reproducibility of results are
 performed for a set of reference configurations included in `tests/pytest/config`.
 Please read the README in `tests/pytest` for more information about these regression
 tests and how to update the reference checksums.
 
-# Interactive development of serialized model in Docker
+# Serialization
+
+The model can be compiled for the generation of serialize data which can be used for unit-testing individual components of the model. For more information see the documentation [here](tests/serialized_test_data_generation/README.md).
+
+To build a docker image for serialization use the command
+
+```bash
+make build_serialize
+```
 
-To enter the docker container with the FV3 sources tree mounted to `/FV3/original`,
-run:
+which will create an image where the source files have been pre-processed for serialization. Similarly to default compilation, you can interactively develop
+for serialization using the command
 
-    make enter_serialize
+```bash
+make enter_serialize
+```
 
-When in the docker container, you can reprocess the sources and recompile the model with:
+The original FV3 sources will be mounted to `/FV3/original`. To compile the model for serialization inside the container, you can use the commands
 
-    cd /FV3 && make -C original serialize_preprocess && make libs fv3.exe
+```bash
+cd /FV3
+make clean
+make -C original serialize_preprocess
+make -j8
+```
 
 If you need to be able to see the preprocessed `atmos_cubed_sphere` sources on your
 host filesystem, you can do that with:
 
-    MOUNTS_SERIALIZE="-v $(pwd)/FV3:/FV3/original -v <local dir>:/FV3/atmos_cubed_sphere" make enter_serialize
+```bash
+MOUNTS_SERIALIZE="-v $(pwd)/FV3:/FV3/original -v <local dir>:/FV3/atmos_cubed_sphere" make enter_serialize
```

Comment:
You removed MOUNTS_SERIALIZE from the Makefile and instead explicitly set the mounts, so these instructions won't work anymore.

Reply:
Changed the logic so that this should still work and is slightly more versatile now.